<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>751</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>#9Slide03 Arima Madurai ExtraBo</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(162, 148, 249);</string>
    </property>
    <property name="text">
     <string>FRACTION</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>491</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>#9Slide03 Arima Madurai ExtraBo</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(245, 239, 255)</string>
    </property>
    <property name="title">
     <string>Input:</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>#9Slide03 Arima Madurai ExtraBo</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Input fraction:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditInput">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>301</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>#9Slide03 Arima Madurai ExtraBo</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(205, 193, 255);
</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditQuantity">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>80</y>
       <width>301</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(205, 193, 255);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>#9Slide03 Arima Madurai ExtraBo</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Quantity:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>#9Slide03 Arima Madurai ExtraBo</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Sort from:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditSortFrom">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>120</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>#9Slide03 Arima Madurai ExtraBo</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(205, 193, 255);
</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>120</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>#9Slide03 Arima Madurai ExtraBo</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Sort to:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditSortTo">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>120</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>#9Slide03 Arima Madurai ExtraBo</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(205, 193, 255);
</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>491</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>#9Slide03 Arima Madurai ExtraBo</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(245, 239, 255)</string>
    </property>
    <property name="title">
     <string>Functions:</string>
    </property>
    <widget class="QPushButton" name="pushButtonNew">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>#9Slide03 Arima Madurai ExtraBo</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(205, 193, 255);</string>
     </property>
     <property name="text">
      <string>New</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonSave">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>40</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>#9Slide03 Arima Madurai ExtraBo</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(205, 193, 255);</string>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonRemove">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>40</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>#9Slide03 Arima Madurai ExtraBo</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(205, 193, 255);</string>
     </property>
     <property name="text">
      <string>Remove</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonExit">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>40</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>#9Slide03 Arima Madurai ExtraBo</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(205, 193, 255);</string>
     </property>
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonSort">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>40</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>#9Slide03 Arima Madurai ExtraBo</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(205, 193, 255);</string>
     </property>
     <property name="text">
      <string>Sort</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonFind">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>40</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>#9Slide03 Arima Madurai ExtraBo</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(205, 193, 255);</string>
     </property>
     <property name="text">
      <string>Find</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>241</width>
      <height>311</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>#9Slide03 Arima Madurai ExtraBo</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(245, 239, 255)</string>
    </property>
    <property name="title">
     <string>List Fraction:</string>
    </property>
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>201</width>
       <height>261</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>#9Slide03 Arima Madurai ExtraBo</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(205, 193, 255);</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>380</y>
      <width>231</width>
      <height>311</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>#9Slide03 Arima Madurai ExtraBo</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(245, 239, 255)</string>
    </property>
    <property name="title">
     <string>List Sort Fraction:</string>
    </property>
    <widget class="QListWidget" name="listWidgetSort">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>191</width>
       <height>261</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>#9Slide03 Arima Madurai ExtraBo</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(205, 193, 255);</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>547</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
